components/financial_assistance/app/views/financial_assistance/incomes/index.html.erb

```diff
@@ -1,212 +1,431 @@
+      <div class='left-seprator'>
+        <a href="#has_self_employment_income" data-toggle="modal" data-target="#has_self_employment_income"><%= l10n("not_sure") %></a>
+        <%= render partial: 'financial_assistance/shared/modal_support_text', locals: {key: "has_self_employment_income"} %>
       </div>
     </div>
-
-    </br>
-    <div id="self_employed_incomes">
-      <div class="self-employed-incomes-list incomes-list">
+    <div id="self_employed_incomes" class="my-4 <%= 'hidden' unless @applicant.incomes.self_employment.any? %>">
+      <div class="self-employed-incomes-list incomes-list pl-0 mb-3">
         <% @applicant.incomes.self_employment.each do |income| %>
-          <%= render partial: 'financial_assistance/incomes/self_employed_income', locals: { income: income } %>
+          <%= render partial: 'financial_assistance/incomes/self_employed_income', locals: { income: income }%>
         <% end %>
       </div>
-
-      <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'new-self-income')" id='new-self-income' class="new-income"> <span class="fa-icon fa-stack plus-mr" aria-hidden='true'><i class="fa fa-circle-thin fa-stack-2x"></i><i class="fa fa-plus fa-stack-1x"></i></span></span> <b>Add Another </b> Self-Employed Income </a>
-
-      <div id="hidden-income-form" class='new-income-form hidden'>
-        <%= render partial: 'financial_assistance/incomes/self_employed_income_form', locals: { income: @applicant.incomes.build, new_form: true} %>
+      <div class="add-more-link mt-3">
+        <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'self-employed-income', 'new-income-form')" id="self-employed-income" class='<%= 'hidden' unless @applicant.incomes.self_employment.present? %> new-income new-income-self-employed button mb-4'><%= l10n("faa.other_incomes.add", kind: "Self-Employed") %></a>
       </div>
-    </div>
-    </br>
-
-    <div class="row">
-      <div class="col-xs-12">
-        <div class="back-link">
-        <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-          <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
-        </div>
+      <div id='new-self-employed-income-form' class='new-income-form my-4 hidden'>
+        <%= render partial: 'financial_assistance/incomes/self_employed_income_form', locals: { income: @applicant.incomes.build, new_form: true } %>
       </div>
     </div>
 
     <div id='income-error-popup'></div>
 
-    <!-- Hidden unsaved income modal -->
-    <div class="modal fade" id="unsavedIncomeChangesWarning" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+    <%= render partial: 'financial_assistance/shared/end_date_warning' %>
+
+    <!-- Hidden unsaved benefits modal -->
+    <div class="modal fade" id="unsavedBenefitChangesWarning" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
```

Comment:
this modal does not appear to be showing when there are incomes being edited/created